mainnet/draft_genesis.json

```diff
@@ -0,0 +1,682 @@
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "0"
+        },
+        {
+          "identifier": "sixhours",
+          "start_time": "0001-01-01T00:00:00Z",
+          "duration": "21600s",
+          "current_epoch": "0",
+          "current_epoch_start_time": "0001-01-01T00:00:00Z",
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "0"
+        },
+        {
+          "identifier": "tenseconds",
+          "start_time": "0001-01-01T00:00:00Z",
+          "duration": "10s",
+          "current_epoch": "0",
+          "current_epoch_start_time": "0001-01-01T00:00:00Z",
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "0"
+        },
+        {
+          "identifier": "twelvehours",
+          "start_time": "0001-01-01T00:00:00Z",
+          "duration": "43200s",
+          "current_epoch": "0",
+          "current_epoch_start_time": "0001-01-01T00:00:00Z",
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "0"
+        },
+        {
+          "identifier": "twohours",
+          "start_time": "0001-01-01T00:00:00Z",
+          "duration": "7200s",
+          "current_epoch": "0",
+          "current_epoch_start_time": "0001-01-01T00:00:00Z",
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "0"
+        },
+        {
+          "identifier": "week",
+          "start_time": "0001-01-01T00:00:00Z",
+          "duration": "604800s",
+          "current_epoch": "0",
+          "current_epoch_start_time": "0001-01-01T00:00:00Z",
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "0"
+        }
+      ]
+    },
+    "estaking": {
+      "params": {
+        "stake_incentives": null,
+        "eden_commit_val": "",
+        "edenb_commit_val": "",
+        "max_eden_reward_apr_stakers": "0.300000000000000000",
+        "eden_boost_apr": "1.000000000000000000",
+        "provider_vesting_epoch_identifier": "ten_days",
+        "provider_staking_rewards_portion": "0.250000000000000000"
+      },
+      "staking_snapshots": []
+    },
+    "evidence": {
+      "evidence": []
+    },
+    "feegrant": {
+      "allowances": []
+    },
+    "feeibc": {
+      "identified_fees": [],
+      "fee_enabled_channels": [],
+      "registered_payees": [],
+      "registered_counterparty_payees": [],
+      "forward_relayers": []
+    },
+    "genutil": {
+      "gen_txs": []
+    },
+    "gov": {
+      "starting_proposal_id": "1",
+      "deposits": [],
+      "votes": [],
+      "proposals": [],
+      "deposit_params": null,
+      "voting_params": null,
+      "tally_params": null,
+      "params": {
+        "min_deposit": [
+          {
+            "denom": "uelys",
+            "amount": "100000000"
+          }
+        ],
+        "max_deposit_period": "1209600s",
+        "voting_period": "432000s",
```

Comment:
@avkr003 this means the voting period in mainnet will last 5 days, I believe what is standard is 48 hrs

Reply:
Standard is 5 days
https://www.mintscan.io/osmosis/parameters
https://www.mintscan.io/cosmos/parameters